packages/mui-material/src/Backdrop/Backdrop.test.js

```diff
@@ -15,11 +15,13 @@ describe('<Backdrop />', () => {
     refInstanceof: window.HTMLDivElement,
     muiName: 'MuiBackdrop',
     testVariantProps: { invisible: true },
-    testLegacyComponentsProp: true,
     slots: {
       root: {
         expectedClassName: classes.root,
       },
+      transition: {
+        testWithElement: null,
+      },
     },
     skip: [
```

Comment:
We can do:

```js
skip: [
    'componentProp',
    'componentsProp',
    // react-transition-group issue
    'reactTestRenderer',
]
```

We can remove `slotPropsCallback` as it should be supported now through `useSlot`